classes/class-p4-master-site.php

```diff
@@ -439,18 +439,23 @@ public function set_custom_allowed_attributes_filter( $allowedposttags ) {
 
 		// Allow img and the following attributes.
 		$allowedposttags['img'] = [
-			'alt'    => true,
-			'class'  => true,
-			'id'     => true,
-			'height' => true,
-			'hspace' => true,
-			'name'   => true,
-			'src'    => true,
-			'srcset' => true,
-			'sizes'  => true,
-			'width'  => true,
-			'style'  => true,
-			'vspace' => true,
+			'alt'         => true,
+			'class'       => true,
+			'id'          => true,
+			'height'      => true,
+			'hspace'      => true,
+			'name'        => true,
+			'src'         => true,
+			'srcset'      => true,
+			'sizes'       => true,
+			'width'       => true,
+			'style'       => true,
+			'vspace'      => true,
+
+			// Required for lazy loading
```

Comment:
This is the only thing I added, the rest is just because of the whitespace change.